components/brave_wallet_ui/components/shared/solana-transaction-instruction/solana-transaction-instruction.tsx

```diff
@@ -220,14 +233,34 @@ const AddressParamValue = ({
 
   // render
   return (
-    <CopyTooltip
-      key={pubkey}
-      text={pubkey}
-      tooltipText={formattedValue === pubkey ? undefined : pubkey}
-      isAddress
-      position='left'
-    >
-      <AddressText>{formattedValue}</AddressText>
-    </CopyTooltip>
+    <>
+      {lookupTableIndex &&
+        <AddressText isBold={true}>
+          {getLocale('braveWalletSolanaAddressLookupTableAccount')}
+        </AddressText>
+      }
+      <CopyTooltip
+        key={pubkey}
+        text={pubkey}
+        tooltipText={
+          formattedValue === pubkey
+            ? undefined
+            : pubkey
+        }
+        isAddress
+        position='left'
+      >
+        <AddressText>{formattedValue}</AddressText>
+      </CopyTooltip>
+      {
+        lookupTableIndex &&
```

Comment:
`0` is treated as false. use `lookupTableIndex !== undefined`

Reply:
Fixed in https://github.com/brave/brave-core/pull/17542/commits/e3febc26a3c28f3edc2bb2ae387d3ca8f731ac7c